flare/struc.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, cell: 'ndarray', species: Union[List[str], List[int]],
         self.vec2 = self.cell[1, :]
         self.vec3 = self.cell[2, :]
 
+        # Compute the max cutoff for sweep = 1.
```

Comment:
Without other context, this comment isn't clear ('What is sweep?'). Perhaps something like "Compute the max cell cutoff for later use of periodic images"?

Reply:
Comment clarified, hopefully it makes more sense now.